scripts/teamcity-migrate.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+set -ex
+
+GIT_ID=$(git rev-parse --short=7 HEAD)
+GIT_BRANCH=$(git symbolic-ref --short HEAD)
+REGISTRY=docker.montagu.dide.ic.ac.uk:5000
+NAME=orderlyweb-migrate
+
+TAG=$REGISTRY/$NAME
+COMMIT_TAG=$REGISTRY/$NAME:$GIT_ID
+BRANCH_TAG=$REGISTRY/$NAME:$GIT_BRANCH
+DB=$REGISTRY/montagu-db:$GIT_ID
+
+docker build \
+       --tag $COMMIT_TAG \
+       --tag $BRANCH_TAG \
+       -f migrations/Dockerfile \
+       .
+
+# run this first to avoid a spurious pull error message
+docker push $COMMIT_TAG
+docker push $BRANCH_TAG
+
+# test if we can run the migrations using the registry image - the following is otherwise the same as test-migrate
+docker pull $COMMIT_TAG
+
+docker pull docker.montagu.dide.ic.ac.uk:5000/orderly:master
+
```

Comment:
I wasn't really sure what the right thing to do here would be. The easiest way to get hold of an orderly db that we can apply our migrations to seems to be to spin up the orderly demo, but maybe there's a better approach?